platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/offline/OfflineRegionDefinition.java

```diff
@@ -0,0 +1,65 @@
+package com.mapbox.mapboxsdk.offline;
+
+import com.mapbox.mapboxsdk.geometry.LatLngBounds;
+
+/**
+ * An offline region defined by a style URL, geographic bounding box, zoom range, and
+ * device pixel ratio.
+ *
+ * Both minZoom and maxZoom must be ≥ 0, and maxZoom must be ≥ minZoom.
+ *
+ * maxZoom may be ∞, in which case for each tile source, the region will include
+ * tiles from minZoom up to the maximum zoom level provided by that source.
+ *
+ * pixelRatio must be ≥ 0 and should typically be 1.0 or 2.0.
+ */
+public class OfflineRegionDefinition {
```

Comment:
Sounds good, this is done in https://github.com/mapbox/mapbox-gl-native/commit/a15175c46e98c4d2cdf53c94180ab0f6edc2e1d3. I took a note on the JNI side (it doesn't affect the higher level API), that part can be addressed when we have more than one definition and https://github.com/mapbox/mapbox-gl-native/issues/4043 is up and running.